frontend/catalyst/jax_primitives_utils.py

```diff
@@ -359,6 +363,26 @@ def transform_named_sequence_lowering(jax_ctx: mlir.LoweringRuleContext, pipelin
                     dynamic_options={},
                 )
                 target = apply_registered_pass_op.result
+
+                try:
+                    # pylint: disable=import-outside-toplevel
+                    from pennylane.compiler.python_compiler.pass_api import (
+                        is_xdsl_pass,
+                    )
+
+                    if is_xdsl_pass(_pass.name):
+                        uses_xdsl_passes = True
+                        apply_registered_pass_op.operation.attributes["xdsl_pass"] = (
+                            ir.UnitAttr.get()
+                        )
+                except ImportError:
```

Comment:
If the error you're trying to catch is for when xDSL is not installed, it would be a `ModuleNotFoundError`, not an `ImportError`.

```suggestion
                except ModuleNotFoundError:
```